package exception;

public class TryFinally {
	public static void main(String[] args) {
		try {
			
		}finally {
			System.out.println("Default Block");
		}
	}
}